app/services/TransferService.scala

```diff
@@ -69,7 +70,10 @@ trait TransferService {
       cache <- cachingService.getUpdateRelationshipCachedData
       _ <- validateTransferorAgainstRecipient(recipientData, cache)
       (recipientRecord, taxYears) <- getRecipientRelationship(transferorNino, recipientData)
-      _ <- cachingService.saveRecipientRecord(recipientRecord, recipientData, taxYears.getOrElse(Nil))
+      validYears <- Future {
```

Comment:
What is the purpose of the future block here?

Reply:
change without Future